pkg/framework/aries/framework.go

```diff
@@ -128,6 +130,21 @@ func WithInboundTransport(inboundTransport transport.InboundTransport) Option {
 	}
 }
 
+// WithTransportReturnRoute injects transport return route option to the Aries framework. Acceptable values - "none",
+// "all" or "thread". RFC - https://github.com/hyperledger/aries-rfcs/tree/master/features/0092-transport-return-route
```

Comment:
Does thread actually work at a global level? Also I don't see return_route_thread attribute :).

> thread: Send all messages matching the cryptographic key and thread specified in the return_route_thread attribute.

Reply:
We can intercept `Thread` at inbound layer as it's a decorator. Currently planning to use senderKey for all to map it with incoming conection. For thread, we can map senderKey-threadID combo.